src/content-feature.js

```diff
@@ -222,4 +246,17 @@ export default class ContentFeature {
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     update () {
     }
+
+    /**
+     * Register a string flag that will be added to page breakage reports
+     * @param {string} flag
+     */
+    addDebugFlag (flag = '') {
```

Comment:
If we're not intending to use this flag code path immediately I would prefer we remove it.

The code instead could set itself to be a noop even (not sure if that's a weird convention and hard to follow) but instead it could check a boolean state instead of a set.